script/Ciel/Day 7/07_01_CIEL07_7.txt

```diff
@@ -440,40 +445,41 @@ The air of provocativeness he had let out earlier is now nowhere to be seen.
 [sha:b46504f8b14742662f7be26b7d914ab61fbe9dbd]{
 -- Page 612, Offset 16279.
 -- マーリオゥの<指|・>が止まる。
+// Kinda weird italics in EN imo. Debating deleting.
 Mario's %{i}fingers%{/i} stop.
 }
 [sha:dad5300cda08e72462a7182c68ff77ef5d592203]{
 -- Page 612, Offset 16280.
 -- 彼はふざけた言葉を吐いた代行者を振り返ろうとして、ソレが掛け値のない真実である事に気がついた。
-// maybe banter is the wrong word
-Just as he is about to turn around and spit out some banter,
- he realizes her words are the unvarnished truth.
+Just as he's about to turn and spit out some pithy retort,
+ he realizes that her words are no idle threat.
 }
 [sha:f3f5b04a156cd6e8d6fd37f2b8f4b354d31524d2]{
 -- Page 613, Offset 16281.
 -- 少年を空中に浮かせている<糸|モノ>はもう一本しかない。
 // this might just be the world's most useless ruby. Good job, Nasu.
-The strings which had been keeping the boy afloat have been reduced to a single one.
+The boy is now dangling in the sky by only a single thread.
```

Comment:
I actually think this simplification impedes the understanding of the scene. This line is trying to explain Mario's previously unexplained floating.